### DISCUSSION TITLE
Desktop client

### DISCUSSION THREAD

**Discussion by trymeouteh** [breezy-weather/breezy-weather#1188] [Ideas]
### Describe your idea

Compile the kotlin codebase into a Windows, MacOS and Linux app.

Certain feature will be different on desktop such as...

- Tray Icon
- No live wallpaper feature

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.5 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Needs more modularization to split the logic part and Android UI-specific. The desktop UI remains to be done entirely.

This will not be actively worked on by anyone, but any change made necessary to make a desktop version possible will be merged into the main code. The desktop UI part will have to be hosted on a different GitHub repo.
